webpack.config.babel.js

```diff
@@ -126,18 +146,37 @@ const config = {
   }
 };
 
-if (ENV === 'production') {
-  config.plugins.push(
-    new webpack.optimize.UglifyJsPlugin({
-      compressor: {
-        pure_getters: true,
-        unsafe: true,
-        unsafe_comps: true,
-        screw_ie8: true,
-        warnings: false
+
+
+const configNode = {
+  ...config,
+  name: 'node-library',
+  output: {
+    library: 'Onfido',
+    libraryTarget: 'umd',
+    path: `${__dirname}/lib`,
+    publicPath: '/',
+    target: 'web',
+    filename: 'index.js'
+  },
+  module: {
+    loaders: [
+      ...baseLoaders,
+      {
+        test: /\.(less|css)$/,
+        loader: [
+          'style-loader',
+          `css?sourceMap=${CSS_MAPS}`,
+          `postcss`,
+          `less?sourceMap=${CSS_MAPS}`
+        ].join('!')
       }
-    })
-  )
+    ]
+  },
+  plugins: [
+    ...basePlugins
+  ].concat(ENV==='production' ? [new webpack.optimize.OccurenceOrderPlugin()] : []),
+  devServer: undefined
 }
 
-export default config
+export default [config,configNode]
```

Comment:
yeah, I guess they are more accurate, but neither truly encapsulates the difference between the two. Is there any name for a bundle like the one in dist? And a name for a bundle which is imported through require and npm like the one in lib? Since those are the actual differences between the two, it would be nice to have a word for each..